todo-list/build/manifest.json

```diff
@@ -0,0 +1,11 @@
+{
+	"name": "Todo List",
+	"description": "Hackathon project",
+	"version": "1.0",
+	"manifest_version": 2,
+	"permissions": ["management", "history", "storage"],
+	"chrome_url_overrides": {
+		"newtab": "index.html"
+	},
+	"content_security_policy": "script-src 'self' 'sha256-GgRxrVOKNdB4LrRsVPDSbzvfdV4UqglmviH9GoBJ5jk='; object-src 'self'"
```

Comment:
Another solution to this could be to add a .env file with INLINE_RUNTIME_CHUNK=false
OR add   "build": "INLINE_RUNTIME_CHUNK=false react-scripts build"
 to your package.json

Reply:
ooooOOOOH this is a game changer for me! thank you lots